purchases/src/test/java/com/revenuecat/purchases/PurchasesTest.kt

```diff
@@ -125,6 +125,11 @@ class PurchasesTest {
         "{'identifier': '\$rc_monthly','platform_product_identifier': '$stubProductIdentifier'}" +
         "]}]," +
         "'current_offering_id': '$stubOfferingIdentifier'}"
+    private val noOfferingsResponse = "{'offerings': [" +
```

Comment:
doesn't this have an offering in it? it has no packages...

Reply:
Fixed, thanks.